### DISCUSSION TITLE
Changing mac address

### DISCUSSION THREAD

**Discussion by sranijymnij** [pgale/lineage_15.1_Installation_TSV#9] [Q&A] [answered]
The addition to the instructions states that if you plan to have several such devices, you need to change the Mac address in one file. Tell me how to do this? Can't root the app Change Mac.

**Comment by pgale** [comment, marked as answer]
You shouldn't need to. Have you checked the MAC addresses? I took out the instructions on how to do it as it wasn't needed and it didn't work anyway. I've not come across need to actually do it, neither has anyone else. I'll double check the bit that says it's needed as that shouldn't be there any more.

**Comment by pgale** [comment]
Closed as no reply and you shouldn't need to change the mac address.
